### PR TITLE
Add AsciiTest (#38)

### PR DESCRIPTION
**Pull request by Pen-y-Fan**
Added requested test for AsciiCommand

### CODE REVIEW

**Comment by programarivm** [comment]
Thanks for sending the PR.

Let me just point out two things:

1. Please make sure to read the [Contributing Guidelines](https://medium.com/geekculture/how-to-contribute-to-chesslab-cca73fefaf70) concerning the git branch name convention; for example, this PR should be named `issue/38-unit-test-the-ascii-command`. However, let's review this PR for the time being.

2. `validate_captures` should be replaced with `validate_ascii` and `validate_captures_foo` with `validate_ascii_foo`

Keep it up and please let us know if you have any questions.

We are here to help and practice multiple skills.

File: tests/unit/Command/AsciiTest.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+use ChessServer\Command\AsciiCommand;
+use ChessServer\Tests\Unit\CommandTestCase;
+
+class AsciiTest extends CommandTestCase
+{
+    /**
+     * @test
+     */
+    public function validate_captures()
```
Comment (programarivm) [inline tests/unit/Command/AsciiTest.php:13]: This function should be renamed to `validate_ascii`.

File: tests/unit/Command/AsciiTest.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            AsciiCommand::class,
+            self::$parser->validate('/ascii')
+        );
+    }
+
+    /**
+     * @test
+     * @expectedException ChessServer\Exception\ParserException
+     */
+    public function validate_captures_foo()
```
Comment (programarivm) [inline tests/unit/Command/AsciiTest.php:25]: This one should be renamed to `validate_ascii_foo()`.

Comment (Pen-y-Fan) [reply tests/unit/Command/AsciiTest.php]: Sorry for my mistake with the method names, I've fixed them now

File: tests/unit/Command/AsciiTest.php (hunk 1 above)
Comment (Pen-y-Fan) [reply tests/unit/Command/AsciiTest.php]: Sorry for my mistake with the method names, I've fixed them.

**Pull request merged**